Ampli Doc Header Update (#238)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sdks/android/index.md

This is the official documentation for the Amplitude Analytics Android SDK.

!!!deprecated "Legacy SDK"

>     This SDK is legacy and only continue to receive bug fixes until deprecated. Checkout and upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations and more.

Reviewer: ```suggestion
    This is a legacy SDK and will only receive bug fixes until deprecation. Upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations, and more.
```

Author: Changed for android and other SDKs that has Legacy block
